packages/arb-validator-core/message/data.go

```diff
@@ -24,16 +24,16 @@ import (
 	"github.com/offchainlabs/arbitrum/packages/arb-util/value"
 )
 
-func bytesToIntValues(val []byte) []*big.Int {
-	var ints []*big.Int
+func bytesToValues(val []byte) []value.Value {
+	var ints []value.Value
 	for i := 0; i < len(val); i += 32 {
 		remaining := len(val) - i
 		if remaining < 32 {
 			data := [32]byte{}
 			copy(data[:], val[i:])
-			ints = append(ints, new(big.Int).SetBytes(data[:]))
+			ints = append(ints, value.NewIntValue(new(big.Int).SetBytes(data[:])))
 		} else {
-			ints = append(ints, new(big.Int).SetBytes(val[i:i+32]))
+			ints = append(ints, value.NewIntValue(new(big.Int).SetBytes(val[i:i+32])))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 86 characters, out of limit 80](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)